eo-parser/src/main/resources/org/eolang/parser/optimize/abstracts-float-up.xsl

```diff
@@ -81,13 +95,17 @@ SOFTWARE.
       </xsl:if>
     </xsl:for-each>
   </xsl:function>
-  <xsl:function name="eo:ancestors">
-    <xsl:param name="object" as="element()"/>
-    <xsl:for-each select="$object/ancestor-or-self::o[eo:abstract(.)]">
-      <xsl:sort data-type="number" select="position()" order="descending"/>
-      <xsl:copy-of select="."/>
-    </xsl:for-each>
```

Comment:
@volodya-lombrozo I don't quite understand why this part was removed?

Reply:
@Graur Because we don't use that function anywhere. The tests prove it.